setup.cfg

```diff
@@ -13,3 +13,9 @@ include_trailing_comma = true
 known_first_party = cms, menus
 multi_line_output = 5
 not_skip = __init__.py
+
+
+[options]
+install_requires =
+    Django>=2.2
```

Comment:
This should probably match what I did here;
https://github.com/django-cms/django-cms/commit/c6c278497a29247954868c617ddab717c864867d